bin/apisix

```diff
@@ -380,6 +380,12 @@ http {
         }
         {% end %}
 
+        location /apisix/aggregate {
```

Comment:
wow, that is wrong. we can use the plugin way to implement this.

please take a look at the plugin `prometheus`: https://github.com/apache/incubator-apisix/blob/master/apisix/plugins/prometheus.lua#L48

Reply:
ok, i will learn about plugin's api first.